app/controllers/users_controller.rb

```diff
@@ -0,0 +1,44 @@
+class UsersController < ApplicationController
+  def show
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to user_path(@user)
+    else
+      respond_with(@user)
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if user.update(user_params)
+      redirect_to user_path(user)
+    else
+      respond_with(user)
+    end
```

Comment:
Could you do this instead?
```
user.update(user_params)

respond_with(user)
```

A different question: why are you using `.update` instead of `.update_attributes`?

Reply:
Yes, that works thank you!
Sorry, I misunderstood in class when we discussed this and changed it.